src/core/src/op/unsqueeze.cpp

```diff
@@ -94,10 +94,7 @@ bool evaluate_unsqueeze(const Node* node,
     out->set_shape(out_shape);
 
     using namespace ov::element;
-    return IfTypeOf<i32, i64, u32, u64, f16, f32, f64, bf16>::apply<Evaluate>(element_type,
-                                                                              arg0,
-                                                                              out,
-                                                                              shape_size(out_shape));
+    return IfTypeOf<i32, i64, u32, u64, f32>::apply<Evaluate>(element_type, arg0, out, shape_size(out_shape));
```

Comment:
side note: `Unsqueeze` is a sort of `Reshape`. Maybe it is more meaningful to unify their evaluates. The code doesn't need to be template here.